LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, ERGS
```

Comment:
I think the copyright line should be different here. The license should be attributed to ARFC or you. What is ERGS?

Reply:
I'll look at the other licenses in the group repository, but I got this from the d3ploy repo. So we might want to revisit that one too.

Reply:
I'll look at the other licenses in the group repository, but I got this from the d3ploy repo. So we might want to revisit that one too.

Reply:
Updated the name, but we should also update the d3ploy license.